### ISSUE TITLE
Konsta config Cannot find module './config/config-extend.js'

### ISSUE THREAD

**Issue by rlora** [konstaui/konsta#77]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://gitlab.com/rlora

### Bug description

Will provide a reproduction repo if needed. Konsta `1.0.0` is packaging the config files in a nested `config` folder causing Tailwind build to fail.

I fixed the issue based on `0.11.1` structure and performed the following workaround.

```
$ npm install
$ cd node_modules/konsta
$ cp config/config/* config/.
```

### Expected Behavior

```
npx tailwindcss -i theme/input.css -o src/lib/theme.css
Done in 538ms.
```

### Actual Behavior

```
npx tailwindcss -i theme/input.css -o src/lib/theme.css

Error: Cannot find module './config/config-extend.js'
Require stack:
- /Users/rolando/Aramcheck/jalasoft/jala-one/traveler/pocs/2022-09-konsta-sveltekit/node_modules/konsta/config.js
- /Users/rolando/Aramcheck/jalasoft/jala-one/traveler/pocs/2022-09-konsta-sveltekit/tailwind.config.cjs
- /Users/rolando/Aramcheck/jalasoft/jala-one/traveler/pocs/2022-09-konsta-sveltekit/node_modules/tailwindcss/lib/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/rolando/Aramcheck/jalasoft/jala-one/traveler/pocs/2022-09-konsta-sveltekit/node_modules/konsta/config.js:2:22)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
```

### Konsta UI version

1.0.0

### Platform/Target and Browser Versions

macOs

### Validations

- [X] Follow our [Code of Conduct](https://github.com/konstaui/konsta/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://konstaui.com).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Konsta UI issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by nolimits4web** [comment]
fixed in 1.0.1

**Comment by rlora** [comment]
Thanks a lot @nolimits4web! Tested 1.0.1 and is working as expected.

**Comment by Jayke770** [comment]
> fixed in 1.0.1

This error also occurred in version 3.3.3.
![image](https://github.com/konstaui/konsta/assets/59675743/5d4aec64-2f3c-4d43-9ed5-f610031289ab)
